README.md

```diff
@@ -104,7 +104,7 @@ val parsedJson = parsed.toJson
 Running the Examples
 --------------------
 
-All examples are stored at [examples folder][examples-folder]. There are 
```

Comment:
Are you removing those spaces on purpose, or is it an IDE that does it? 

Does it really matter if there is an extra space in the end? We need to come into agreement on how to do it, because I'm upset when see such changes in commits.